src/main/java/hudson/plugins/parameterizedtrigger/BuildTriggerConfig.java

```diff
@@ -486,7 +487,7 @@ private void reportSchedulingError(@Nonnull Run<?, ?> run, @Nonnull Job<?, ?> jo
      */
     private List<List<AbstractBuildParameters>> getDynamicBuildParameters(AbstractBuild<?,?> build, BuildListener listener) throws DontTriggerException, IOException, InterruptedException {
         if (configFactories == null || configFactories.isEmpty()) {
-            return ImmutableList.of(ImmutableList.of());
+            return Collections.singletonList(Collections.emptyList());
```

Comment:
Unnecessary wrapping
Both `java.util.Collections.EmptyList` and `java.util.Collections.SingletonList` are immutable.

```suggestion
            return Collections.emptyList();
```

Reply:
nope... look again... this is a list of lists..  and the default was a list containing an empty list...

I had what you suggested first time round and spent a day bashing my head working out why the tests where all failing :)

Changing what the code behaves is out of scope of this PR :)

Reply:
Easy to get blinded by wrappings around wrappings :D